starter_code/Css/Styles.css

```diff
@@ -0,0 +1,92 @@
+header{
+    display: flex;
+    background: black;
+    justify-content: space-around;
+    height:100px;
+    border-bottom: yellow solid 3px;
+}
+
+.logo{
+    color:yellow;
+   font-size: 40px;
+}
+nav ul li{
+    display: inline;
+    color: white;
+     padding: 5px;
+}
+.taste{
+    text-decoration:line-through;
+    color:gray;
+    text-align: center;
+}
+main{
+    position: relative;
+    background-image:url(../images/coworking.jpg);
+    height:500px;
+
+}
+
+a{
+    color: white;
+}
+
+.section1{
+    text-align: center;
+    background-color: gainsboro;
+    height: 50px;
+    font-size: 25px;
+    padding-top: 10px;
+    font-weight: bold;
+}
+
+.section2{
+ display: flex; 
+ flex-direction:row;
+justify-content: space-around;
```

Comment:
If you want your content display nicely try to replace this property with `align-items: center;`
and, add this selector to make the content of the section in the center.
```
.section2 div{
  width: 100%;
  text-align: center;
}
```